test_fixtures/missing/pubspec.lock

```diff
@@ -0,0 +1,59 @@
+# Generated by pub
```

Comment:
Note: These `pubspec.lock` files need to be present in order to run test fixtures, but we don't need to keep the actual versions up to date. Only the names of the dependencies matter.

Reply:
Should we exclude them from the `.gitignore`, then, and add this information as a comment?